src/shared/components/palette/fixed-list/write-fixed-list-field.html

```diff
@@ -1,16 +1,14 @@
-<div class="form-group" [ngClass]="{'form-group-error': !fixedListControl.valid && fixedListControl.touched}">
+<div class="form-group">
 
   <label [for]="id()">
     <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
     <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
-    <span class="error-message" *ngIf="fixedListControl.errors && fixedListControl.touched">{{fixedListControl.errors | ccdFirstError}}</span>
```

Comment:
we have removed `ccdFirstError` pipe. If you look at FirstErrorPipe, this pipe is used to provide common standard error messages. For example if the field is required, the common error message should be:
`'This field is required'`
What error message are we getting now?

Reply:
we need to ask QAing around error messages